orchestration/dap_orchestration/config/resources/dataflow_beam_runner/global.yaml

```diff
@@ -1,3 +1,5 @@
+# todo: hardcode this to be relative to the solid that’s configuring it by getting the dir from the configured solid
+working_dir: ../../
```

Comment:
@aherbst-broad You mentioned that I should be configuring this path relative to the solid that is configuring it by getting the dir from the configured solid via __file__.

Would I just be doing that in the test_pipelines code or any scratch files I create for testing?
Here's [the example you linked](https://github.com/DataBiosphere/hca-ingest/blob/f6703e4f48dc3a13642d1ffe1a48ba4a14e3da13/orchestration/dagster_orchestration/hca_orchestration/tests/test_pipelines.py#L14-L16
test_pipelines.py)

Reply:
We'd need to do it in code, not via YAML. I'd need to think about how to that will play with our `preconfigure` mechanism. I think this is fine for now.